docs/components/connectors/out-of-the-box-connectors/amazon-dynamodb.md

```diff
@@ -204,19 +210,36 @@ For the **Table** operation type, the following input data is required:
+| [Primary key components](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html#DDB-UpdateItem-request-Key)        | map       | Yes      | A map of attribute names to `AttributeValue` objects, representing the primary key of the item to update.                 |
+| [Key attributes](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html#DDB-UpdateItem-request-AttributeUpdates)   | map       | Yes      | A map of attribute names to `AttributeValue` objects, representing the attributes to update.                              |
+| [Attribute action](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html#DDB-UpdateItem-request-AttributeUpdates) | dropdown  | No       | Dropdown option for each attribute to be updated, allowing selection between "PUT" (add or replace) and "DELETE" (remove) |
 
 **Response**
 
 | Property                                                                                                                                            | Data type | Description                                                                                                                                                                                                                 |
 | --------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [UpdateItemOutcome](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.Methods.html#DynamoDBMapper.Methods.updateItem) | object    | An object representing the outcome of the `UpdateItem` operation. The `UpdateItemOutcome` object contains the updated attributes of the item, as well as other metadata about the operation, such as the consumed capacity. |
 
+## Request example
+
+| Section        | Field                  | Description                                                                                                                        | Example value                                                               |
+| -------------- | ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| Operation      | Category               | Choose the category of the operation to be performed.                                                                              | Item                                                                        |
```

Comment:
Are all of the fields referenced with this same casing in the UI? If not, these should likely be sentence case.

Reply:
It looks based on this screenshot that everything is sentence-case, so everything here should also be sentence-case. For example, "Table name" and "Key attributes".

Am I reading this correctly? Can I assist?